tools/example-showcase/src/main.rs

```diff
@@ -580,6 +581,10 @@ header_message = \"Examples ({})\"
                                 .skip(1)
                                 .collect::<PathBuf>()
                                 .display(),
+                            match to_show.shader_paths.len() {
+                                0 => "".to_string(),
+                                _ => format!("\nshader_code_paths = {:?}", to_show.shader_paths),
+                            },
```

Comment:
and then this one can print an empty array if no shader is needed
```suggestion
                            format!("{:?}", to_show.shader_paths),                       
```